src/Application/Features/Assessments/DTOs/V1/Pathways/WellbeingAndMentalHealth/H12.cs

```diff
@@ -0,0 +1,12 @@
+namespace Cfo.Cats.Application.Features.Assessments.DTOs.V1.Pathways.WellbeingAndMentalHealth;
+
+public class H12() : SingleChoiceQuestion("How often do you feel stressed?", [
```

Comment:
Ran through the scoring/assessment implementation- this question is currently missing answers and doesn't let you proceed further and needs addressing.